apps/src/code-studio/teacher.js

```diff
@@ -176,10 +176,12 @@ function renderViewAsToggle(element) {
 function renderTeacherPanelSections(element) {
   ReactDOM.render(
     <Provider store={getStore()}>
-      <SectionSelector
-        reloadOnChange={true}
-        alwaysShow={true}
-      />
+      <span style={{margin: 10}}>
+        <SectionSelector
+          reloadOnChange={true}
+          alwaysShow={true}
```

Comment:
Interesting - `alwaysShow` seems like a bit of logic that should maybe live outside of `SectionSelector`, if only because if I tried to reuse it I wouldn't immediately guess that I needed to provide this property or it wouldn't be visible sometimes.

I'm also in favor of shorthand for true boolean props:
```jsx
<SectionSelector reloadOnChange alwaysShow/>
```